.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -58,8 +85,8 @@ jobs:
         env:
           NODE_ENV: production
 
-  test:
-    name: Test
+  web-test-runner:
```

Comment:
Had to rename our testing jobs similar to how I did our NPM scripts now that we have (below) two jobs using the same test runner.